interpreter/cling/lib/Interpreter/IncrementalJIT.cpp

```diff
@@ -478,7 +478,7 @@ IncrementalJIT::IncrementalJIT(
   ErrorAsOutParameter _(&Err);
 
   LLJITBuilder Builder;
-  Builder.setDataLayout(TM->createDataLayout());
+  Builder.setDataLayout(m_TM->createDataLayout());
```

Comment:
this commit was already backported in https://github.com/root-project/root/pull/12337!